helm/charts/determined/templates/master-ingress.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-prod
+    ingress.kubernetes.io/force-ssl-redirect: "true"
+    ingress.kubernetes.io/ssl-redirect: "true"
+    traefik.ingress.kubernetes.io/redirect-entry-point: https
+  labels:
+    app.kubernetes.io/name: determined-master-{{ .Release.Name }}
+  name: determined-master-{{ .Release.Name }}
+spec:
+  rules:
+  - host: determined.{{ .Release.Namespace }}.ord1.ingress.coreweave.cloud
```

Comment:
@salanki does ingress region change? i.e. ord1 becomes lga1 etc. I don't think i've ever used anything other than ord1.

Reply:
Yes, it does. See this ingress definition that can be used mostly unmodified: https://github.com/coreweave/kube-charts/blob/master/charts/grafana/templates/grafana-ingress.yaml

Reply:
Let me know if we should make it static @salanki

Reply:
Not sure what you mean @rtalaricw, you should make it like it is in the link I sent